src/main/cpp/events/sampleevent.cpp

```diff
@@ -420,8 +420,8 @@ void SampleEvent::mixBuffer( AudioBuffer* outputBuffer, int bufferPosition,
     float fEventStart        = ( float ) _eventStart;
     float fEventEnd          = ( float ) getEventEnd();
     float fMinBufferPosition = ( float ) minBufferPosition;
-    float fMaxBufferPosition = ( float ) maxBufferPosition * _playbackRate;
-    float fLoopOffset        = ( float ) loopOffset * _playbackRate;
+    float fMaxBufferPosition = _playbackRate < 1.F ? maxBufferPosition / _playbackRate : maxBufferPosition * _playbackRate;
+    float fLoopOffset        = _playbackRate < 1.F ? loopOffset / _playbackRate        : loopOffset * _playbackRate;
 
```

Comment:
This was broken since forever. When reducing (slowing down) playback, the events wouldn't play for their full range.